src/arch/riscv/utility.hh

```diff
@@ -308,7 +308,7 @@ elem_mask(const T* vs, const int index)
 
 template<typename T>
 inline int
-elem_mask_vlseg(const T* vs, const int elem, const int num_fields)
+elem_mask_vseg(const T* vs, const int elem, const int num_fields)
```

Comment:
(nit) Why you need to check the function name?

Reply:
The thing is that I created `elem_mask_vlseg` when implementing the load operations (vlseg -> vector load segmented) but the store operations (vsseg) also use the same function, so I wanted to make the name more generic and cover both (vseg -> vector segmented). 

In any case, let me know if you would prefer another naming for it.